devicemapper/thin_pool_watcher_test.go

```diff
@@ -46,6 +46,18 @@ func TestRefresh(t *testing.T) {
 			expectedError: true,
 		},
 		{
+			name: "no existing reservation - ok with minimum # of fields",
+			dmsetupCommands: []fake.DmsetupCommand{
+				{"status", "0 75497472 thin-pool 65 327/524288 14092/589824 -", nil}, // status check
```

Comment:
so when i run dmsetup status <pool> , i can get values like:

0 37740544 thin-pool 247 1065/262144 66667/294848 - rw no_discard_passdown queue_if_no_space

can we have an example with a larger number of fields?